third_party/ruby/ruby-2.6.BUILD

```diff
@@ -19,13 +19,14 @@ config_setting(
     values = {"host_cpu": "k8"},
 )
 
-
```

Comment:
Are the changes in these two ruby BUILD files mostly just buildifier formatting changes, or did you have to do a lot of work to get ruby to build under new bazel specifically?

Reply:
There was only one significant change to each file, adding the `toolchains`  attribute to each  `run_configure` genrule:

https://github.com/sorbet/sorbet/pull/2444/files#diff-30490048ed9b3e2a33af671e6ad05b20R72-R75